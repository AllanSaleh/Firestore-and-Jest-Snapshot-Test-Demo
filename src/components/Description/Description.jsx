import React from "react";

export const Description = () => {
  return <div>Kishi</div>;
};
