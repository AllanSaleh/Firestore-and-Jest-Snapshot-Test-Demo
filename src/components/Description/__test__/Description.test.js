import renderer from "react-test-renderer";
import { Description } from "../Description";

test("Description component renders correctly", () => {
  const tree = renderer.create(<Description />).toJSON();
  expect(tree).toMatchSnapshot();
});
