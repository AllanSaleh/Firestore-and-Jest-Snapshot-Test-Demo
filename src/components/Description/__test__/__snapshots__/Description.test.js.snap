// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Description component renders correctly 1`] = `
<div>
  Kishi
</div>
`;
